src/mscorlib/src/System/Globalization/TimeSpanParse.cs

```diff
@@ -419,6 +420,9 @@ internal Boolean FullHMSFMatch(TimeSpanFormat.FormatLiterals pattern)
             private const int MaxLiteralTokens = 6;
             private const int MaxNumericTokens = 5;
 
+            internal TimeSpanToken numbers0, numbers1, numbers2, numbers3, numbers4; // MaxNumbericTokens = 5
+            internal ReadOnlySpan<char> literals0, literals1, literals2, literals3, literals4, literals5; // MaxLiteralTokens=6
```

Comment:
@jaredpar, per our conversation, this is an example of where fixed buffer support for non-primitive types would be really handy.  You can see here:
https://github.com/dotnet/coreclr/pull/13619/commits/4ad7260d872b2356262337b15f854bddd7bbe01d#diff-e581a221cef607379517b80100253877R486
having to need to implement our own lame "indexer", and then all of the changes starting around:
https://github.com/dotnet/coreclr/pull/13619/commits/4ad7260d872b2356262337b15f854bddd7bbe01d#diff-e581a221cef607379517b80100253877R927
that are necessary to target the individual fields, making for a less-than-ideal porting experience from strings to spans.
cc: @MadsTorgersen